lips/lip-6.mdx

```diff
@@ -0,0 +1,253 @@
+* [`add_validator_and_reconfigure`](https://github.com/libra/libra/blob/master/language/stdlib/transaction_scripts/add_validator_and_reconfigure.move) - to add Validator Owner to the Validator Set with its current config and trigger reconfiguration
+* [`remove_validator_and_reconfigure`](https://github.com/libra/libra/blob/master/language/stdlib/transaction_scripts/remove_validator_and_reconfigure.move) - to remove Validator Owner from the Validator Set and trigger reconfiguration
+
+The following transaction scripts can be run by the Validator Owner:
+* [`set_validator_operator`](https://github.com/libra/libra/blob/master/language/stdlib/transaction_scripts/set_validator_operator.move) - to designate the Validator Operator
+
+The following transaction scripts can be run by the Validator Operator:
+* [`set_validator_config_and_reconfigure`](https://github.com/libra/libra/blob/master/language/stdlib/transaction_scripts/set_validator_config_and_reconfigure.move) - to change values of the Validator Config resource and update the config in the Validator Set (aborts if the Validator Owner is not in the Validator Set)
+
+The following script relevant to validator's operations can also be run by any LibraAccount's owner:
+* [`rotate_authentication_key`](https://github.com/libra/libra/blob/master/language/stdlib/transaction_scripts/rotate_authentication_key.move) - to rotate the authentication key of the Validator Owner Account
+
+# Workflows
+
+This section details the workflows or common scenarios related to Validator Set management.
+## Validator Owner and Validator Operator accounts creation
+LibraRoot (through the Association Operation Services, "AOS") creates Validator Owner and Validator Operator accounts. Currently, Validator Owners are not maintaining ownership of their Libra accounts. The LibraRoot account sends Validator Owners' transaction on their behalf via AdminScripts. The Validator Owner's account address is derived from the name of the Validator Owner, the auth-key is all-zeroes vector.n The LibraRoot creates the Validator Owner's account by sending the `create_validator_account` transaction script.
+
+Validator Operators on the other hand maintain full ownership of their Libra accounts. Validator Operators should contact the AOS off-chain and communicate their authentication keys as well as the names of their organizations written in snake case (lowercase, where spaces are replaced with underscores). The authentication key is a 32 elements byte array, derived from the public verification key. The protocol currently supports Ed25519 and MultiEd25519 signature schemes. The account's address for Validator Operators is derived in a usual way.  The LibraRoot creates the Validator Owner's account by sending the `create_validator_operator_account` transaction script.
```

Comment:
yeah so my reading is we should probably have 3 paragraphs

Reply:
Hmm.. I wanted to say how account addresses are defined is an implementation detail... but that might be too hand wavy. I want to think about this more, but this okay.

i do think this means we should have 3 paragraphs, generalized account creation process and then how it differs for the two.

Reply:
Sounds good, I tried to make it 3 paragraphs.